src/armor_layers.cpp

```diff
@@ -317,8 +317,8 @@ void player::sort_armor()
             trim_and_print( w_sort_left, drawindex + 1, offset_x, left_w - offset_x - 3,
                             tmp_worn[itemindex]->damage_color(),
                             tmp_worn[itemindex]->type_name( 1 ).c_str() );
-            mvwprintz( w_sort_left, drawindex + 1, left_w - 3, c_ltgray, "%3d ml",
-                       to_milliliter( tmp_worn[itemindex]->get_storage() ) );
+            right_print( w_sort_left, drawindex + 1, 0, c_ltgray, "%3d", 
+                            to_milliliter( tmp_worn[itemindex]->get_storage() ) );
```

Comment:
No, those are spaces.

The line 321 changed (4 spaces added), I guess because my editor (VSCode) wanted this line to align with line 319.

I can fix this if you want.